dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -23,13 +23,11 @@ def __init__(self, name, options=None):
         :param name: Name of data
         :type name: String
         """
-
-        self.options = options
+        if options and not isinstance(options, CategoricalOptions):
+            raise ValueError("options must be of type CategoricalOptions.")
```

Comment:
I don't think a user would understand this and is it even possible to error out?

Reply:
shouldn't happen, but who knows. How should I make this error message more clear? I think if a user gets this error, they will know what it means imo